packages/shorebird_cli/README.md

```diff
@@ -78,6 +78,21 @@ shorebird logout
 ✓ Logging out of shorebird.dev (1ms)
 ```
 
+### Account
+
+To see information about your Shorebird account, use the `shorebird account` command.
```

Comment:
Overall the implementation looks great but do we want `shorebird account` to be responsible for returning account info? How will this look if we also add support for editing account info?

What are your thoughts on tweaking this to be `shorebird account info` to align with the existing commands like `shorebird apps create` `shorebird apps delete`, etc. or maybe even `shorebird whoami` if the goal is to provide info about the current user? Again the implementation is awesome I'm mainly trying to make sure I understand where we're going and how this would evolve in the future.

Reply:
Sorry! Merged this before I saw this comment (interpreted https://github.com/shorebirdtech/shorebird/pull/231#discussion_r1159155010 as an LGTM). Working on a quick outline of the subscription mechanism now. I could see this command becoming less useful or maybe an amalgamation of `shorebird apps list` and `shorebird subscription`.